packages/mdc-checkbox/mdc-checkbox.scss

```diff
@@ -20,116 +20,5 @@
 // THE SOFTWARE.
 //
 
-@import "@material/ripple/common";
-@import "@material/ripple/mixins";
 @import "./mixins";
-@import "./variables";
-
-// postcss-bem-linter: define checkbox
-
-.mdc-checkbox {
-  @include mdc-checkbox-base_;
-  @include mdc-ripple-surface;
-  @include mdc-states($mdc-checkbox-baseline-theme-color);
-  @include mdc-ripple-radius-unbounded;
-}
-
-@at-root {
```

Comment:
I should point out that one change I did make was to remove this `@at-root`. It didn't seem necessary, and it would prevent people from doing something like:

```scss
@include mdc-checkbox();

.my-app-alternate-theme {
  // ... change some theme variables ...
  @include mdc-checkbox(color);
}
```

Correct me if I'm wrong though.

Reply:
Yeah, I noticed this, and realized that it seemed unnecessary, since it wasn't nested under anything and afaik `@include` can be done top-level anyway. We might've gotten a little carried away when we were trying to reduce specificity.